src/main/java/ro/andreiciortea/yggdrasil/template/acta4h/Curtains.java

```diff
@@ -0,0 +1,60 @@
+package ro.andreiciortea.yggdrasil.template.acta4h;
+
+import io.vertx.core.http.HttpMethod;
+
+import ro.andreiciortea.yggdrasil.template.annotation.RequestMapping;
+import ro.andreiciortea.yggdrasil.template.annotation.Artifact;
+import ro.andreiciortea.yggdrasil.template.annotation.ObservableProperty;
+import ro.andreiciortea.yggdrasil.template.annotation.RdfAddition;
+
+
+@Artifact(type = "Curtains", additions =
```

Comment:
Here type should be an IRI defined in some ontology, so instead of "Curtains" it would rather be something like "http://example.org/#Curtains". Same applies for the other templates.

Reply:
resolved